raiden/tests/utils/genesis.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8
+from raiden.settings import GAS_LIMIT_HEX
+GENESIS_STUB = {
+    'config': {
+        'homesteadBlock': 0,
+        'eip150Block': 0,
+        'eip150Hash': '0x0000000000000000000000000000000000000000000000000000000000000000',
+        'eip155Block': 0,
+        'eip158Block': 0,
+    },
+    'nonce': '0x0',
+    'mixhash': '0x0000000000000000000000000000000000000000000000000000000000000000',
+    'difficulty': '0x1',
+    'coinbase': '0x0000000000000000000000000000000000000000',
+    'timestamp': '0x00',
+    'parentHash': '0x0000000000000000000000000000000000000000000000000000000000000000',
+    'extraData': '0x' + 'raiden'.encode('hex'),
+    'gasLimit': GAS_LIMIT_HEX,
+    # add precompile addresses with minimal balance to avoid deletion
```

Comment:
These addresses are reserved for precompiled contracts, see e.g. go-ethereums [current list of implemented precompiles](https://github.com/ethereum/go-ethereum/blob/94c71c171f6c4de1f6f041bf1aadef2a377e10a3/core/vm/contracts.go#L39) and the [scheme they use for reserving the addresses](https://github.com/ethereum/go-ethereum/blob/fff16169c64a83d57d2eed35b6a2e33248c7d5eb/cmd/puppeth/wizard_genesis.go#L114).

I can change it to whatever you want, but the wording sounds reasonable to me..?